backend/sirius-components-diagrams/src/main/java/org/eclipse/sirius/components/diagrams/components/NodeComponentProps.java

```diff
@@ -83,6 +85,10 @@ public String getParentElementId() {
         return this.parentElementId;
     }
 
+    public List<String> getPreviousNodeIds() {
```

Comment:
This does not contain the id of the previous nodes.

Reply:
Changed to better name